project_event/models/activity_template.py

```diff
@@ -26,9 +28,40 @@ class ActivityTemplate(models.Model):
         'activity.category',
         string='Category',
     )
+    room_id = fields.Many2one(
+        'resource.calendar.room',
+        string='Room',
+    )
     task_template_ids = fields.One2many(
         'task.template',
         'activity_template_id',
         string='Tasks Template',
     )
     notes = fields.Text(string='Notes')
+
+    @api.multi
+    def action_clear(self):
+        self.ensure_one()
+        self.task_template_ids.unlink()
+
+    @api.multi
+    def action_initialize(self):
+        self.action_clear()
+        task = {
+            'room_id': self.room_id.id,
+            'resource_type': 'room',
+            'name': self.name,
+            'activity_template_id': self.id,
+        }
+        self.env['task.template'].create(task)
+        for instrument in self.room_id.instruments_ids:
```

Comment:
Exception for: no instruments in a room

Reply:
So there is no ids...this will throw an error

Reply:
@eilst if there is no instrument_ids it will not enter in the for clause